sdk-generate-entity-models-maven-plugin/src/main/java/com/hpe/adm/nga/sdk/generate/GenerateModels.java

```diff
@@ -136,7 +136,18 @@ private boolean entityShouldNotBeGenerated(String name) {
     }
 
     private Map<String, String> generateLists(Octane octane) throws IOException {
-        final Collection<EntityModel> listNodes = octane.entityList("list_nodes").get().addFields("name", "list_root", "id", "logical_name").execute();
+        final Collection<EntityModel> listNodes = new ArrayList<>();
+
+        int offset = 0;
+        Collection<EntityModel> fetchedListNodes =
+                octane.entityList("list_nodes").get().addFields("name", "list_root", "id", "logical_name").limit(1000).offset(offset).execute();
```

Comment:
The request can be refactored into a GetEntities and reused below with only execute